crates/ruff_python_formatter/src/snapshots/ruff_python_formatter__tests__black_test__attribute_access_on_number_literals_py.snap

```diff
@@ -0,0 +1,148 @@
+---
+source: crates/ruff_python_formatter/src/lib.rs
+expression: snapshot
+input_file: crates/ruff_python_formatter/resources/test/fixtures/black/simple_cases/attribute_access_on_number_literals.py
+---
+## Input
+
+
+```py
+x = 123456789 .bit_count()
```

Comment:
You can override the filetype for .py.snap in your JetBrains IDE to get markdown rendering

![image](https://user-images.githubusercontent.com/1203881/219748195-d1f71cc4-fc07-41c3-bcb5-0c92af14d470.png)